tests/unit/worker_manager/slurm_batch_worker_manager_test.py

```diff
@@ -15,7 +15,7 @@ def test_base_command(self):
             partition='some_partition',
             worker_executable='cl-worker',
             worker_idle_seconds='888',
-            worker_tag='some_tag',
+            worker_tag='sometag',
```

Comment:
Why remove the underscores here?

Reply:
worker tags have to be alphanumeric, so these tags aren't actually valid. This is just a mocked test / I haven't added the functionality to take multiple tags to the workermanager yet, but I'm planning on doing that in a subsequent PR.

Reply:
I opened a PR at https://github.com/codalab/codalab-worksheets/pull/3350 that builds upon this one